src/mmw/apps/geoprocessing_api/tasks.py

```diff
@@ -229,3 +230,112 @@ def collect_climate(results):
             'categories': categories
         }
     }
+
+
+@shared_task
+def analyze_terrain(result):
+    """
+    Starts a job to produce summary statistics for elevation and slope in a
+    given area.
+
+    TODO Where does the NED and Slope data come from?
```

Comment:
cc @arottersman -- do you know the answer to this?

Reply:
NED data comes from NHDPlus V2 NEDSnapshot, and Slope data is from NED (National Elevation Dataset)